src/walk-manifest.js

```diff
@@ -47,14 +49,26 @@ const mediaGroupPlaylists = function(mediaGroups) {
   return playlists;
 };
 
-const parseManifest = function(content) {
+const parseM3u8Manifest = function(content) {
   const parser = new m3u8.Parser();
 
   parser.push(content);
   parser.end();
   return parser.manifest;
 };
 
+const parseMpdManifest = function(content, srcUrl) {
+  global.DOMParser = DOMParser;
```

Comment:
since `mpd-parser` uses `window.DOMParser` and `window` is `global` in node we add a global here to deal with that. Not ideal, and perhaps we should just use `xmldom` in `mpd-parser` and take it out of the browser bundle, but for now this works.